identity/webapp/tsconfig.json

```diff
@@ -1,46 +1,31 @@
 {
   "compilerOptions": {
     "composite": true,
-    "baseUrl": "src",
+    "baseUrl": ".",
     "lib": [
       "dom",
       "dom.iterable",
       "es2020"
     ],
-    "typeRoots": [
-      "../../node_modules/@types",
-      "./node_modules/@types"
-    ],
     "target": "es2018",
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "module": "commonjs",
     "declaration": true,
     "moduleResolution": "node",
     "noEmit": true,
-    "strict": true,
-    "sourceMap": true,
+    "strict": false,
```

Comment:
Why is this?

Reply:
I think this was required to run Next and the other APIs through the same server, but would need confirmation from @jamieparkinson

Reply:
Yeah, there's some stuff in `common` that isn't `strict` (the other apps aren't either) and thought it was a bit much to change that here